interface/super/edit_list.php

```diff
@@ -233,6 +234,7 @@
                 "'".($row['maxseq']+1)."',".
                 "'1', '0')"
                 );
+    $list_id = $newlistID;
```

Comment:
When saving a new list, this allows the empty rows to load up by default instead of having to re-select the new list